bin/site-inspector

```diff
@@ -1,15 +1,29 @@
 #!/usr/bin/env ruby
 
 require_relative "../lib/site-inspector"
+
 require "oj"
 
 domain = ARGV[0]
+http_mode = (ARGV[1] == "--http")
```

Comment:
I looked long and hard for a good, simple Ruby command line option parser, and eventually figured this was easier. It could be that this mode is so different that it becomes its own binary or something.